wolfcrypt/test/test.c

```diff
@@ -53418,7 +53648,7 @@ static int myCEKwrapFunc(wc_PKCS7* pkcs7, byte* cek, word32 cekSz, byte* keyId,
 
     switch (keyWrapAlgo) {
         case AES256_WRAP:
-            ret = wc_AesKeyUnWrap(p7DefKey, sizeof(p7DefKey), cek, cekSz,
+            ret = myAesKeyUnWrap(p7DefKey, sizeof(p7DefKey), cek, cekSz,
```

Comment:
Don't run this test/code if `HAVE_AES_KEYWRAP` is not defined. We don't need to reimplement Aes key wrapping here for this scenario. No value at all to duplicate code...